postcss.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  plugins: {
+    'postcss-import': {
+      root: __dirname,
+    },
+    'postcss-mixins': {},
+    'postcss-each': {},
+    'postcss-cssnext': {}
+  },

```

Comment:
Just published `beta.6`. It should work now because we get rid of the composes mess